freebites/product
# 39 sorting manuel keiji

# Description

Implemented sorting for Most Recent, Closest to you, Perishables and Non-Perishables

Changed front-end for the Perishables to include Perishables and Non-Perishables in a separate section from Sort.

Modified the appContext to include a 'perish' item.

## Issues

Resolve #39 

## Screenshots

![image](https://github.com/freebites/product/assets/147024363/13245552-d027-4dd0-a543-761f6bc29229)

## Test

1. Open app on Expo 
2. Go to Home page
3. Click sidebar to access sorting and filter modal
4. Click on Most Recent to and check if the posts are ordered in the most recently posted order
5. Repeat the same process for Perishables and Non Perishables and check if the posts are filtered correctly
6. Toggle the Perishables and Non Perishables and make sure both filters aren't turned on simultaneously
7. Reload to make sure the changes are still there
